twitchio/ext/eventsub/models.py

```diff
@@ -420,17 +420,17 @@ class ChannelRaidData(EventData):
     -----------
     raider: :class:`twitchio.PartialUser`
         The person initiating the raid
-    reciever: :class:`twitchio.PartialUser`
+    receiver: :class:`twitchio.PartialUser`
         The person recieving the raid
     viewer_count: :class:`int`
         The amount of people raiding
     """
 
-    __slots__ = "raider", "reciever", "viewer_count"
+    __slots__ = "raider", "receiver", "viewer_count"
 
     def __init__(self, client: EventSubClient, data: dict):
         self.raider = _transform_user(client, data, "from_broadcaster_user")
-        self.reciever = _transform_user(client, data, "to_broadcaster_user")
+        self.receiver = _transform_user(client, data, "to_broadcaster_user")
```

Comment:
This constitutes a breaking change. If you'd like to add it as a secondary attribute you can do that instead